platforms/r3-corda/configuration/samples/network-minikube.cordav2.yaml

```diff
@@ -0,0 +1,449 @@
+    password: "docker_password"
+  
+  # Remote connection information for doorman and networkmap (will be blank or removed for hosting organization)
+  orderers:
+    - orderer:
+      type: doorman
+      uri: http://<NODE_IP_ADDRESS>:30030
+      certificate: /home/blockchain-automation-framework/platforms/r3-corda/configuration/build/corda/doorman/tls/ambassador.crt
+    - orderer:
+      type: networkmap
+      uri: http://<NODE_IP_ADDRESS>:30040
+      certificate: /home/blockchain-automation-framework/platforms/r3-corda/configuration/build/corda/networkmap/tls/ambassador.crt
+  
+  # Allows specification of one or many organizations that will be connecting to a network.
+  # If an organization is also hosting the root of the network (e.g. doorman, membership service, etc),
+  # then these services should be listed in this section as well.
+  organizations:
+    # Specification for the 1st organization. Each organization maps to a VPC and a separate k8s cluster
+    - organization:
+      name: supplychain
+      country: UK
+      state: London
+      location: London
+      subject: "CN=DLT Root CA,OU=DLT,O=DLT,L=London,C=GB"
+      type: doorman-nms-notary
+      external_url_suffix: develop.local.com
+  
+      cloud_provider: minikube   # Options: aws, azure, gcp
+  
+      # Kubernetes cluster deployment variables. The config file path and name has to be provided in case
+      # the cluster has already been created.
+      k8s:
+        region: "cluster_region"
+        context: "minikube"
+        config_file: "/home/blockchain-automation-framework/build/config"
+
+      # Hashicorp Vault server address and root-token. Vault should be unsealed.
+      # Do not check-in root_token
+      vault:
+        url: "vault_addr"
+        root_token: "vault_root_token"
+
+      # Git Repo details which will be used by GitOps/Flux.
+      # Do not check-in git_password
+      gitops:
+        git_ssh: "gitops_ssh_url"         # Gitops ssh url for flux value files like "ssh://git@github.com/hyperledger-labs/blockchain-automation-framework.git"
+        branch: "gitops_branch"           # Git branch where release is being made
+        release_dir: "gitops_release_dir" # Relative Path in the Git repo for flux sync per environment. 
+        chart_source: "gitops_charts"     # Relative Path where the Helm charts are stored in Git repo
+        git_push_url: "gitops_push_url"   # Gitops https URL for git push like "github.com/hyperledger-labs/blockchain-automation-framework.git"
+        username: "git_username"          # Git Service user who has rights to check-in in all branches
+        password: "git_password"          # Git Server user password
+        email: "git_email"                # Email to use in git config
+        private_key: "path_to_private_key"          # Path to private key file which has write-access to the git repo
+
+      # Cordapps Repository details (optional if cordapps jar are store in a repository)
```

Comment:
we are not building jars beforehand, so this can be removed. If needed we can use manual approach.